src/lsdb/loaders/hipscat/margin_catalog_loader.py

```diff
@@ -0,0 +1,24 @@
+import hipscat as hc
+
+from lsdb.catalog.margin_catalog import MarginCatalog
+from lsdb.loaders.hipscat.abstract_catalog_loader import AbstractCatalogLoader
+
+
+class MarginCatalogLoader(AbstractCatalogLoader[MarginCatalog]):
+    """Loads an HiPSCat MarginCatalog"""
+
+    def load_catalog(self) -> MarginCatalog:
```

Comment:
For abstract methods, the docstring is inherited from the super class when rendering docs (and presumably in hover text in various tools). I'd prefer to have the single docstring in the ABC if it's just duplicated in subclasses.